cypress/e2e/failedWithdrawalDueToInsufficientFunds.spec.cy.js

```diff
@@ -0,0 +1,51 @@
+/// <reference types='cypress' />
+
+describe('Bank app', () => {
```

Comment:
1. There are too long file names. In the file name, we should highlight the feature we test, and that's all. Check and fix everywhere
2. Each describe should have unique name (we will see them in test report)